.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.11
+  - 1.13
```

Comment:
should we bump this to 14 or 15?

Reply:
want to try to be as backwards as possible for public clients, 1.13 brings along new error handling, anything in 14 or 15 we are or that you would like to use now or in the near future?